neo/CMakeLists.txt

```diff
@@ -66,17 +72,16 @@ if (CMAKE_SYSTEM_PROCESSOR MATCHES "(x86)|(X86)|(amd64)|(AMD64)")
 	set(CPU_OPTIMIZATION "-mmmx -msse -msse2" CACHE STRING "Which CPU specific optimitations should be used beside the compiler's default?")
 endif()
 
-if(CMAKE_SYSTEM_PROCESSOR MATCHES "((powerpc|ppc)64le)|(mips64)")
-	option(USE_INTRINSICS "Compile using intrinsics (e.g mmx, sse, msse2)" OFF)
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "(x86)|(X86)|(amd64)|(AMD64)")
+	option(USE_INTRINSICS_SSE "Compile using SSE intrinsics (e.g mmx, sse, msse2)" ON)
 else()
-	option(USE_INTRINSICS "Compile using intrinsics (e.g mmx, sse, msse2)" ON)
+	option(USE_INTRINSICS_SSE "Compile using SSE intrinsics (e.g mmx, sse, msse2)" OF)
```

Comment:
I think it was a typo and already fixed it.

Reply:
Cool, thanks!
BTW, can we undo the renaming of the option; some distributions (like Debian on i386*) might rely on the old name and could introduce bugs if they miss the change.
(It could be just /me forgetting about the change when I prepare the next release.)

* i386 baseline for Debian is without SSE support...

Reply:
I don't want to change it because of further upcoming intrinsinc changes that aren't SSE. There was already a pull request for AVX intrinsics and with Apple's move to ARM I can think of Neon intrinsics sooner or later.

https://github.com/RobertBeckebans/RBDOOM-3-BFG/pull/452  still needs to be resolved with a USE_INSTRINSICS_AVX option.

Reply:
Ok, makes sense if making it more granular;
However I guess SSE and AVX are mutually exclusive, aren't they? If Im not mistaken, SSE is x86 only and AVX is arm only?

Reply:
PS: Possibly more work, but runtime-detection could be also an option: Google has a library for that: https://github.com/google/cpu_features

Reply:
> However I guess SSE and AVX are mutually exclusive, aren't they? If Im not mistaken, SSE is x86 only and AVX is arm only?

Nope, AVX is also x86, it's like a successor of SSE4.
ARMs SIMD extensions are called NEON.